scripts/local-node-setup.sh

```diff
@@ -57,7 +64,6 @@ echo "📖️ Deploying Vectis contracts and running tests..."
 
 cd ../cli
 
-npm ci
-npm test
+npm run test:ci 
```

Comment:
```npm ci```  is necessary in order to install node dependencies, this command is similar to ``` npm install``` except it's meant to be used in automated environments, it loads dependencies from package.json and [other benefits](https://docs.npmjs.com/cli/v8/commands/npm-ci)